libraries/namespaces.raml

```diff
@@ -0,0 +1,7 @@
+#%RAML 1.0 Library
+annotationTypes:
+  namespace:
+    description: |
+      this annotation type allows you to associate unique identifier with a RAML document.
+    type: string
+(namespace): http://raml.org/vocabularies/annotations/core
```

Comment:
So as well as I can guess you are voting for explicitly requiring it to be url. (As for me I am almost 100% sure that it should be urls, but probably would prefer to write this in comment for this version of the library)

>Do we have a type uri already or do you need to define it?
I have this type in other library which I am not contributing yet. Do you think that we should include it to this PR?

Regards,
Pavel